docs/user-guide/admin-cli.md

```diff
@@ -93,35 +99,100 @@ optional arguments:
   -h, --help  show this help message and exit
 ```
 
+### Update
+Update existing tenant. This command allows to update "Description", "Default_datastore" or rename an existing tenant.
```

Comment:
This command allows to update "Description" and "Default_datastore" fields, or rename an existing tenant.